cmd/parse/parse.go

```diff
@@ -100,11 +100,6 @@ func StartParsing(ctx *Context) error {
 	// Listen for and trap any OS signal to gracefully shutdown and exit
 	trapSignal(ctx)
 
-	if cfg.ParseGenesis {
-		// Add the genesis to the queue if requested
-		exportQueue <- 0
-	}
-
```

Comment:
Is this really necessary? This will avoid parsing the genesis **always**. Instead, I think we should let the user decide whether they want to parse it or not. The BDJuno `fix genesis` command can then simply re-parse the genesis instead. This way we preserve the current Juno behavior and still allow to re-parse the genesis whenever we want

Reply:
ok!